components/script/dom/dommatrixreadonly.rs

```diff
@@ -553,6 +576,25 @@ impl DOMMatrixReadOnlyMethods for DOMMatrixReadOnly {
 
         DOMPoint::new(&self.global(), x, y, z, w)
     }
+
+    // https://drafts.fxtf.org/geometry-1/#dom-dommatrixreadonly-tofloat32array
+    #[allow(unsafe_code)]
+    unsafe fn ToFloat32Array(&self, cx: *mut JSContext) -> NonNull<JSObject> {
+        let vec: Vec<f32> = self.matrix
+            .borrow().to_row_major_array().iter().map(|&x| x as f32).collect();
+        rooted!(in (cx) let mut array = ptr::null_mut::<JSObject>());
+        let _ = Float32Array::create(cx, CreateWith::Slice(&vec), array.handle_mut());
```

Comment:
We should call unwrap() here, or we could end up with a null pointer.

Reply:
ok will fix